spec/amp-var-substitutions.md

```diff
@@ -76,6 +76,28 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
 
+### SOURCE_URL
+
+Use the special string `SOURCE_URL` to add the source URL of the current document to the URL
+
+For instance:
+```html
+<amp-pixel src="https://foo.com/pixel?href=SOURCE_URL"></amp-pixel>
+```
+may make a request to something like `https://foo.com/pixel?href=https%3A%2F%2Fpinterest.com%2F`.
+
+The source URL is extracted from the proxy URL if the document is proxied or otherwise the original document is returned.
+
+### SOURCE_HOST
+
+Use the special string `SOURCE_HOST` to add the source URL's host of the current document to the URL
+
+For instance:
+```html
+<amp-pixel src="https://foo.com/pixel?host=SOURCE_HOST"></amp-pixel>
+```
+may make a request to something like `https://foo.com/pixel?host=pinterest.com`.
```

Comment:
In this section, clarify with more description about how this has to do with proxy URLs and include a more specific example. See comments regarding SOURCE_URL.